tests/runner.py

```diff
@@ -590,18 +590,17 @@ def prep_ll_file(self, output_file, input_file, force_recompile=False, build_ll_
     # Recompiling just for dfe in ll_opts is too costly
 
     def fix_target(ll_filename):
-      if LLVM_TARGET == ASM_JS_TARGET:
-         return
       with open(ll_filename) as f:
         contents = f.read()
       if LLVM_TARGET in contents:
         return
+      asmjs_target = 'asmjs-unknown-emscripten'
```

Comment:
may as well delete the code from here, as the return on the line before will always happen?

Reply:
Na.  Sadly will have a bunch of test `.ll` files checked in with the fastcomp triple.  We can look at changing that later.

Reply:
Oh, I see...

We can likely remove most of those .ll tests, btw.